### ISSUE TITLE
2.0.0-alpha.4 won't run, claiming configuration cache issues

### ISSUE THREAD

**Issue by ursusursus** [detekt/detekt#9390]
```
./gradlew :base:detektMain 
Calculating task graph as no cached configuration is available for tasks: :detektAllDebug

1 problem was found storing the configuration cache.
- Task `:base:detektMain` of type `dev.detekt.gradle.Detekt`: error writing value of type 'org.gradle.api.internal.DefaultNamedDomainObjectCollection$ExistingNamedDomainObjectProvider'

See the complete report at file:///Users/foo/AndroidStudioProjects/bar/build/reports/configuration-cache/1xoaogjjy49iu8afavslw14qh/2gajzs3sqqb97arov5mrqryi3/configuration-cache-report.html

[Incubating] Problems report is available at: file:///Users/foo/AndroidStudioProjects/bar/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Configuration cache state could not be cached: field `provider` of `org.gradle.internal.serialize.codecs.core.ProviderBackedFileCollectionSpec` bean found in field `__pluginClasspath__` of task `:base:detektMain` of type `dev.detekt.gradle.Detekt`: error writing value of type 'org.gradle.api.internal.DefaultNamedDomainObjectCollection$ExistingNamedDomainObjectProvider'
> Could not resolve all files for configuration ':base:detektPlugins'.
   > Could not find dev.detekt:ktlint-repackage:2.0.0-alpha.4.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/dev/detekt/ktlint-repackage/2.0.0-alpha.4/ktlint-repackage-2.0.0-alpha.4.pom
       - https://repo.maven.apache.org/maven2/dev/detekt/ktlint-repackage/2.0.0-alpha.4/ktlint-repackage-2.0.0-alpha.4.pom
     Required by:
         project ':base' > dev.detekt:detekt-rules-ktlint-wrapper:2.0.0-alpha.4

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights from a Build Scan (powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 7s
Configuration cache entry discarded due to serialization error.
```

I simply run the detekt main task and get this error, hasn't happened before

**Comment by cortinico** [comment]
@ursusursus does it work if you disable configuration cache.

> > Could not find dev.detekt:ktlint-repackage:2.0.0-alpha.4.

This is definitely expected as per:
- #9314

I believe the problem is that `detekt-rules-ktlint-wrapper` still has a dependency on `ktlint-repackage` inside the POM file.

**Comment by ursusursus** [comment]
no, same result, i tried nuking the cache, still nothing

**Comment by cortinico** [comment]
Yeah that makes sense. Fix is here:
- https://github.com/detekt/detekt/pull/9391

I'll do an alpha.5 as soon as that lands

**Comment by IRus** [comment]
Just in case, temporary workaround:

```kotlin
detektPlugins(libs.detekt.rules.ktlint.wrapper) {
    exclude(group = "dev.detekt", module = "ktlint-repackage")
}
```
